build/templates/_converters.py.mako

```diff
@@ -324,3 +326,50 @@ def convert_double_each_element(numbers):
 
 
 % endif
+% for enum_name in sorted(helper.filter_codegen_enums(enums)):
+    % if enums[enum_name].get('use_converter', False):
+<%
+    enum_name_in_snakecase = helper.camelcase_to_snakecase(enums[enum_name]['python_name'])
+%>\
+# Getter converter for ${enums[enum_name]['python_name']}
```

Comment:
These generated converters could apply to methods, right?

Reply:
Technically yes, but the codegen logic doesn't handle the usage of enums with converter for function parameters currently.